bin/lib/create.js

```diff
@@ -54,34 +54,12 @@ module.exports.createProject = (project_path, package_name, project_name, option
     fs.ensureDirSync(project_path);
 
     // copy templates/build-res directory ( recursive )
```

Comment:
Maybe, `// copy templates directory to the project path recursively` would be more accurate?

Reply:
👍 Yes, good catch. I didn't notice the comment.

I will update the comment as suggested and also change the wording from `project path` to `platform directory`.

```suggestion
    // / copy templates directory to the platform directory recursively
```

There could be confusion between the project root and the platform's dir with the variable name `project_path`.

To potentially reduce confusion, do you think I should update this variable name, in this PR, to `platform_dir`?